src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -340,18 +342,65 @@ public Page Pop()
             return curTop;
         }
 
+        /// <summary>
+        /// Returns the index of the given page in Navigator.
+        /// The indices of pages in Navigator are basically the order of pushing or inserting to Navigator.
+        /// So a page's index in Navigator can be changed whenever push/insert or pop/remove occurs.
+        /// </summary>
+        /// <param name="index">The index of a page in Navigator.</param>
```

Comment:
Probably 
```/// <returns> some descritpion```

should be added here.

Reply:
Thank you for your comments.
I have added the returns documentation.